server/app_env/lite.py

```diff
@@ -17,4 +17,5 @@
 
 class LocalConfig(local.Config):
   SECRET_PROJECT = ''
-  LITE = True
\ No newline at end of file
+  LITE = True
+  SHOW_DISASTER = True
```

Comment:
in the original config we don't show disaster on lite, did you want to  change that?

Reply:
Removed